torch/cuda/amp/autocast_mode.py

```diff
@@ -144,7 +147,7 @@ def _cast(value, dtype):
         return value.to(dtype) if is_eligible else value
     elif isinstance(value, string_classes):
         return value
-    elif isinstance(value, np.ndarray):
+    elif np is not None and isinstance(value, np.ndarray):
         return value
```

Comment:
nit: was wondering if we can just remove these 2 and add comments on the last default return (since it is a passthrough)?

Reply:
That wouldn't work, since numpy arrays are iterable (i.e. `isinstance(np.ndarray(5), container_abcs.Iterable)` returns True )